src/supervise-daemon/supervise-daemon.c

```diff
@@ -1209,13 +1260,13 @@ int main(int argc, char **argv)
 
 		rc_service_value_set(svcname, "pidfile", pidfile);
 		varbuf = NULL;
-		xasprintf(&varbuf, "%i", respawn_delay);
+		xasprintf(&varbuf, "%zd:%zd", respawn_delay.tv_sec, respawn_delay.tv_nsec);
```

Comment:
Same comment about wrong specifier. Would be better to store string directly, IMO.